### PR TITLE
Ingestion - add confirmation to skip + help button

### PR DESCRIPTION
**Pull request by kpthatsme**
## Changes
[Ticket](https://github.com/PostHog/product-internal/issues/112?notification_referrer_id=MDE4Ok5vdGlmaWNhdGlvblRocmVhZDIyMDM5NTY4NDM6NTk2NTg5MQ%3D%3D&notifications_query=is%3Adone#issuecomment-888507292)
- Add confirmation dialog before skipping without events
- Add escape valve helper links (invite a teammate, ask in slack)

State management was a little tricky - we re-render this component every 1.5 sec to check for new events, so that's why there's code taking component state to determine whether to load a user or not. 

Open to feedback here if there's a better way to have done it!

Screen Shot 2021-07-27 at 6 41 51 PM
Screen Shot 2021-07-27 at 6 41 39 PM
Screen Shot 2021-07-27 at 6 41 46 PM

## Checklist

- [ ] All querysets/queries filter by Organization, by Team, and by User
- [ ] Django backend tests
- [ ] Jest frontend tests
- [ ] Cypress end-to-end tests
- [ ] Migrations are safe to run at scale (e.g. PostHog Cloud) – present proof if not obvious
- [ ] New/changed UI is decent on smartphones (viewport width around 360px)

### CODE REVIEW

File: frontend/src/scenes/ingestion/panels/VerificationPanel.tsx [hunk 1]
```diff
@@ -1,21 +1,117 @@
...
     const { loadUser } = useActions(userLogic)
     const { user } = useValues(userLogic)
     const { setVerify, completeOnboarding } = useActions(ingestionLogic)
     const { index, totalSteps } = useValues(ingestionLogic)
+    const { featureFlags } = useValues(featureFlagLogic)
+    const [isMenuOpen, setIsMenuOpen] = useState(false)
 
     useInterval(() => {
-        !user?.team?.ingested_event && loadUser()
+        if (!user?.team?.ingested_event) {
```
Comment (alexkim205) [inline frontend/src/scenes/ingestion/panels/VerificationPanel.tsx:24]: maybe you can simplify this to `if (!user?.team?.ingested_event && !isMenuOpen)`?

Comment (kpthatsme) [reply frontend/src/scenes/ingestion/panels/VerificationPanel.tsx]: done

**Comment by alexkim205** [comment]
This looks really useful! There's only a few minor bugs:

Dropdown menu disappears if you don't wait for menu to completely unfold. Might be a antd `trigger` prop that you need to change?

https://user-images.githubusercontent.com/13460330/127395360-d80c2907-8721-4237-97dc-d2a18a669935.mov

Have to click confirm twice to activate popover


https://user-images.githubusercontent.com/13460330/127395999-1efa3e09-c463-4aa8-954e-92d3a1adce47.mov

**Comment by kpthatsme** [comment]
> This looks really useful! There's only a few minor bugs:
> 
> Dropdown menu disappears if you don't wait for menu to completely unfold. Might be a antd `trigger` prop that you need to change?
> 
>  Screen.Recording.2021-07-28.at.2.00.39.PM.mov 
> Have to click confirm twice to activate popover
> 
>  Screen.Recording.2021-07-28.at.2.06.06.PM.mov

Thanks @alexkim205 - both of these should be fixed now

**Comment by alexkim205** [comment]
🙌 lgtm!

**Review by alexkim205 — APPROVED** [review]

**Review by samwinslow — APPROVED** [review]

**Pull request merged**